.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/**
+.eggs/**
```

Comment:
How to get something installed there ? It's not created on my system. If it's not used, it shouldn't be listed

Reply:
did you run the setup.py sdist command ?

Reply:
Yes, it only created `dist/`

Reply:
ok so this .eggs migth be the result of a first misconfiguration of my setup.py. 
I will update accordingly.

Reply:
Just ran a setup.py  bdist_egg. just `dist` directory created. So updated in last commit.